docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   spec:
     build: .
-    command: 'bash -c "cd /app/user && sleep 10 && crystal spec"'
+    command: 'bash -c "cd /app/user && sleep 10 && bin/ameba && crystal spec"'
```

Comment:
Also add `crystal tool format --check`?

Reply:
I added it to the docker compose file. I tried to run it in the command line on my own machine but it did not change any file. I tried to read to cli help but it just stated "crystal tool format". Also checking crystal documentation but could not find it. Am i doing something wrong or are all files already formatted correctly. If i change something to be really off that it still does nothing for me. Could you provide me more information regarding to the tool format functionality?

Reply:
Here is a good guide for [Writing Shards](https://crystal-lang.org/docs/guides/writing_shards.html#coding-style) in the official documentation and it mentioned `crystal tool format --check`.
Also you can see more formatter options in `crystal tool format --help`.